src/io/flutter/actions/InstallSdkAction.java

```diff
@@ -31,15 +53,166 @@ String getLinkText() {
     }
   }
 
+  private static class GitCloneAction extends InstallAction {
+    private final FlutterGeneratorPeer myPeer;
+
+    public GitCloneAction(FlutterGeneratorPeer peer) {
+      myPeer = peer;
+    }
+
+    @Override
+    void perform() {
+      @SuppressWarnings("DialogTitleCapitalization") final FileChooserDescriptor descriptor =
```

Comment:
Food for thought: should we hard-code to a specific SDK location? `~/flutter`? I haven't thought about it - and it has drawbacks - but could be a nice simplification here and for users in general.